manifests/java.yml

```diff
@@ -37,7 +37,8 @@ tests/:
             vertx3: v1.12.0
             vertx4: v1.12.0
         test_hardcoded_secrets.py:
-          Test_HardcodedSecrets: missing_feature            
+          Test_HardcodedSecrets:
```

Comment:
You activated nothing here, is it on purpose ?

Reply:
Yes, I 'm waiting to merge the tested feature in dd-trave-java to add to enable the test in springboot, it's almost done so I will wait for that and then update this PR